setup.py

```diff
@@ -124,8 +122,9 @@
         "extra": [
             # For render
             "opencv-python",
+            "pygame",
             # For atari games,
-            "ale-py==0.7.4",
+            "ale-py~=0.8.0",
```

Comment:
0.8.1

Reply:
fixed in the Gymnasium branch (where we rely on shimmy)